src/doc/trpl/installing-rust.md

```diff
@@ -26,48 +26,108 @@ the [64-bit installer][win64] and run it.
 [win32]: https://static.rust-lang.org/dist/rust-1.0.0-beta-i686-pc-windows-gnu.msi
 [win64]: https://static.rust-lang.org/dist/rust-1.0.0-beta-x86_64-pc-windows-gnu.msi
 
+If you would like to build from source, please check out the
+documentation on [building Rust from Source][from source], there is always [the official
+binary downloads][install page] as well. 
+
+[from source]: https://github.com/rust-lang/rust#building-from-source
+[install page]: http://www.rust-lang.org/install.html
+
+If you're on Linux or Mac rust has a tool called ['multirust'](https://github.com/brson/multirust/README.md) for managing
```

Comment:
It's not my place to say but I really think it's quite a lot better than the other command used to install.  If it's stable but not feature complete my vote(do I get a vote?) is to include it.  I think multirust is an amazing tool, since there is not apt-get rust it really is the forerunner for version and package management, at least on linux.  IMHO it's very good.  I will fill the ./multirust_advsetup.md if @brson is alright with it.